crates/optimism/rpc/src/eth/mod.rs

```diff
@@ -226,6 +237,12 @@ where
     fn fee_history_cache(&self) -> &FeeHistoryCache {
         self.inner.eth_api.fee_history_cache()
     }
+
+    async fn suggested_priority_fee(&self) -> Result<U256, Self::Error> {
+        let base_tip = self.inner.eth_api.gas_oracle().suggest_tip_cap().await?;
+        let min_tip = U256::from(self.inner.min_suggested_priority_fee);
+        Ok(base_tip.max(min_tip))
```

Comment:
nice this should do it, for now,

we can still port the entire logic, but this should address low fees already